packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAmountInput/TransferFormAmountInput.tsx

```diff
@@ -42,6 +44,15 @@ const TransferFormAmountInput: React.FC<TProps> = ({ fieldName }) => {
         ? fromAccount?.currencyConfig?.fractional_digits
         : toAccount?.currencyConfig?.fractional_digits;
 
+    const convertedMaxPossibleAmount = useMemo(
+        () => USD_MAX_POSSIBLE_TRANSFER_AMOUNT * (USDExchangeRates?.rates?.[currency ?? 'USD'] ?? 1),
+        [USDExchangeRates?.rates, currency]
+    );
+    const { value: formattedConvertedMaxPossibleAmount } = useInputDecimalFormatter(convertedMaxPossibleAmount, {
+        fractionDigits,
+    });
+    const maxDigits = formattedConvertedMaxPossibleAmount.match(/\d/g)?.length ?? MAX_DIGITS;
```

Comment:
minor: unit tests for this case? e.g. "validates that number of digits in user input isn't bigger than number of digits in maximum value after coversion" 
(though complexity of this case also makes me think if this level of validation precision is actually needed)

Reply:
I would love to see things being simple:
validate max value, min value, maximum decimal points